### PR TITLE
including performance tests

### PR DESCRIPTION
**Pull request by fabricio-aguiar**
ref 5506
https://pulp.plan.io/issues/5506

### CODE REVIEW

File: templates/travis/.travis/performance_test_script.sh.j2 [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+if [ "$TRAVIS_EVENT_TYPE" = "cron" ]; then
+  echo "--- Performance Tests ---"
+  pytest -vv -r sx --color=yes --pyargs --durations=0 pulp_rpm.tests.performance
```
Comment (bmbouter) [inline templates/travis/.travis/performance_test_script.sh.j2]: The `pulp_rpm` needs to be generalized with a jinja template for the plugin name.

**Comment by bmbouter** [comment]
@fabricio-aguiar I left one request, but I'd also like to ask @dkliban to review also before merging.

File: templates/travis/.travis/performance_test_script.sh.j2 (hunk 1 above)
Comment (fabricio-aguiar) [reply templates/travis/.travis/performance_test_script.sh.j2]: thanks, I didn't notice it

File: templates/travis/.travis/performance_test_script.sh.j2 [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+if [ "$TRAVIS_EVENT_TYPE" = "cron" ]; then
+  echo "--- Performance Tests ---"
+  pytest -vv -r sx --color=yes --pyargs --durations=0 {{ plugin_snake }}.tests.performance
```
Comment (daviddavis) [inline templates/travis/.travis/performance_test_script.sh.j2]: :+1:

File: plugin-template [hunk 3]
```diff
@@ -26,6 +26,7 @@ DEFAULT_SETTINGS = {
     'docs_test': True,
     'pydocstyle': True,
     'test_bindings': True,
+    'test_performance': True,
```
Comment (dkliban) [inline plugin-template]: can you change the default to False please?

**Pull request merged**